platform-test-images.yml workflow fails on dependabot prs

see [here](https://github.com/gardenlinux/gardenlinux/actions/workflows/platform-test-images.yml)

all builds in `dependabot/something` branches fail with a message like

```
Writing manifest to image destination
Error: writing manifest "{\"schemaVersion\":2,\"config\":{\"mediaType\":\"application/vnd.oci.image.config.v1+json\",\"digest\":\"sha256:89ab7a703b13370c5e7d2d24bf1b6150244662612b01589f9a4dd20c61acaaf4\",\"size\":4407},\"layers\":[{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:77d95ad2f720643df7ac240bbe1db276c48ffb94515382df6444f1125b6fa82e\",\"size\":34392078},{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:f2b3a19c9045a433010ffa7f1010675ddac02bb9d469707fccaae36b98d6158b\",\"size\":202489733},{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:9513512a5ada8e7af6aa02e48792375bc14a9eb0221b7b223fcaea26d6bf56bc\",\"size\":3620909},{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:52e8a12654c78d76f35d53c87b1e2893f94f2c4d1ff4a4e2dc41a56acca6f846\",\"size\":84140},{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:52133854543b214071b5f89693c3ef05ab03c3be5eb7a19b6150a6f7b4c5fcbe\",\"size\":91318371},{\"mediaType\":\"application/vnd.oci.image.layer.v1.tar+gzip\",\"digest\":\"sha256:f59f9761a27bb5324a5249536e918461f164abd66ca218006dfcbc802bbe2c14\",\"size\":235310032}]}": uploading manifest 6c92af3b2ec9e9e59dbaa65c18f49ff6089444a1 to ghcr.io/gardenlinux/gardenlinux/platform-test-kvm: denied: installation not allowed to Write organization package
make: *** [Makefile:110: push-platform-test-kvm] Error 125
make: Leaving directory '/home/runner/work/gardenlinux/gardenlinux/tests/images'
```

looks like it's an permission issue with the dependabot user?

--- a/.github/workflows/platform-test-images.yml
+++ b/.github/workflows/platform-test-images.yml
@@ -96,13 +96,6 @@ jobs:
           else
             echo "Tag for ${GL_VERSION} already exists in ghcr.io/gardenlinux/platform-test-${{ matrix.platform }} and FORCE_BUILD=false"
           fi
-      # always run push-platform-test
-      - name: make push-platform-test-${{ matrix.platform }}
-        env:
-          # needed to use github cli in bin/garden-version-resolver
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          make -j $(nproc) --directory=tests/images GARDENLINUX_BUILD_CRE=podman push-platform-test-${{ matrix.platform }}
       # after merge to main, run push-release-platform-test
       - name: make push-release-platform-test-${{ matrix.platform }} (after merge to main)
         if: github.event.pull_request.merged == true && github.ref_name == 'main'

